app/styles/modules/popup.scss

```diff
@@ -260,11 +260,14 @@
     margin: 0;
     border: 0;
 }
-/* attach popup */
-.attach-photo {
+/* attach/avatar popup */
+.attach-photo, .avatar-pic {
```

Comment:
.attach-photo,
.avatar-pic {
...
}

Read one more time https://github.com/QuickBlox/CodeStyle-Guidelines-for-Developers/tree/master/web/css